recipes/libzippp/all/conanfile.py

```diff
@@ -0,0 +1,75 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibZipppConan(ConanFile):
+    name = "libzippp"
+    description = "A simple basic C++ wrapper around the libzip library"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ctabin/libzippp"
+    license = "custom"
```

Comment:
```suggestion
    license = "BSD-3-Clause"
```

Reply:
@intelligide the library actually uses a custom license: [LICENCE](https://github.com/ctabin/libzippp/blob/master/LICENCE)
I have asked in an issue if they could switch to a common license instead.
Do you think it's fine if we put BSD3 instead?

Reply:
It's just the BSD-3-Clause ([See here](https://spdx.org/licenses/BSD-3-Clause.html)). GitHub doesn't detect it because the author has added an optional extra line. ;)

Reply:
@intelligide thanks! will fix